docs/extending/plugins-list.md

```diff
@@ -120,3 +120,7 @@
   - `verifyConditions`: Verify the environment variable ```PYPI_TOKEN``` and installation of build tools
   - `prepare`: Update the version in ```setup.cfg``` and create the distribution packages
   - `publish`: Publish the python package to a repository (default: pypi)
+- [semantic-release-helm](https://github.com/m1pl/semantic-release-helm)
+  - `verifyConditions`: Validate configuration and (if present) credentials
+  - `prepare`: Update version and appVersion in ```Chart.yaml```
+  - `publish`: Publish the chart to a registry (if configured)
```

Comment:
changes look good 👍🏼 Just a suggestion: add a link to https://helm.sh/ in your plugin's README, for people (like myself) who don't know what it is but are curious

Reply:
Thank you for merging. There you go:
https://github.com/m1pl/semantic-release-helm/commit/aeef54beee74f78ef293d0c27924cacc957225cf